### DISCUSSION TITLE
What month is named after the Roman god of war?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#881] [Galaxy Trivia] [answered]
**Bonus:** List all months of the original Roman calendar.

**Comment by SANKARAMDAS** [comment]
March
March. March is named after Mars, the Roman god of war. This statue shows him in battle gear. The Roman calendar originally began in March, and the months of January and February were added later, after a calendar reform.



Month Names | Number of Days
Ianuarius - 29
Februarius -28
Martius- 31
Aprilis- 29
Maius- 31
Iunius- 29
Quintilis- 31
Sextilis- 29
September- 29
October- 31
November- 29
December- 29

https://www.timeanddate.com/calendar/roman-calendar.html

**Comment by peterbaccaro** [comment, marked as answer]
The month named after the Roman god of war is **March**. In the ancient Roman calendar, March was the first month of the year, and it was named after Mars, the god of war. Mars was considered a significant deity in Roman mythology, associated with military power, courage, and strength. March was traditionally seen as a time of renewed military campaigns and the beginning of the spring season. Over time, the calendar was modified, and January and February were added, pushing March to the third position in the calendar. However, it still retained its association with Mars.

The original Roman calendar (also known as the pre-Julian calendar or Roman Republican calendar) consisted of ten months. Here is a list of those months:

- Martius (March) - 31 days
- Aprilis (April) - 30 days
- Maius (May) - 31 days
- Junius (June) - 30 days
- Quintilis (July) - 31 days
- Sextilis (August) - 30 days
- September - 30 days
- October - 31 days
- November - 30 days
- December - 30 days

The names of these months are derived from various sources, such as gods, numbers, or agricultural activities. Originally, the Roman calendar only had ten months and consisted of 304 days, with the new year starting in March. Later, two additional months, January and February, were added to align the calendar with the lunar year, resulting in the familiar twelve-month calendar we use today.
